test/new_relic/manticore/instrumentation_test.rb

```diff
@@ -112,7 +96,6 @@ class InstrumentationTest < Minitest::Test
           assert_operator(exclusive_time, :>, 1.0)
 
           assert_metrics_recorded(db_metric => { call_count: 1 })
-          assert_metrics_not_recorded("External/#{external_service}/Manticore/POST" => { call_count: 1 })
```

Comment:
the whole test can be removed as we dont need to care about elasticsearch anymore